R/RemapLog.R

```diff
@@ -0,0 +1,57 @@
+#' Remap the log output of Make_Snapshot to new col names and indexes
+#'
+#' @param table `data.frame` the table of interest from `Make_Snapshot()` lSnapshot log
+#' @param table_name `character` optional argument to define table to remap by name, when the table object name doesn't match
+#'
+#' @importFrom purrr map_chr
+#' @importFrom purrr map_int
+#' @importFrom yaml read_yaml
+#' @importFrom yaml yaml.load
+#' @importFrom cli cli_abort
```

Comment:
Very minor but these can be added on the same line.

```suggestion
#' @importFrom purrr map_chr map_int
#' @importFrom yaml read_yaml yaml.load
#' @importFrom cli cli_abort
```